apps/storybook/src/TabButton.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import { TabButton } from '@asap-hub/react-components';
+
+export default {
+  title: 'Atoms / Navigation / Tab Button',
+};
+
+export const Active = () => <TabButton active={true}>Button</TabButton>;
```

Comment:
This should just be one story called Normal. Active and the text in the button could be knobs